src/Components/Day.vue

```diff
@@ -20,6 +20,10 @@
                     :is-day-selected="isDaySelected"
                     v-for="event in day.events">
             </EventCard>
+
+            <EventModal title="Test" :show.sync="showAddEvent" :day="day">
```

Comment:
Please update the EventModal with the following:

`<EventModal
                    :show.sync="showAddEvent"
                    :day="day">
            </EventModal>`